packages/api-explorer/src/CodeSample.jsx

```diff
@@ -35,7 +37,8 @@ class CodeSample extends React.Component {
         </ul>
         <div className="code-sample-body">
           {examplesWithLanguages.map(example => {
-            return (
+            return [
+              <CopyCode code={example.code} />,
```

Comment:
Does this not make React complain about there not being a `key` on the array items returned?